libiop/algebra/field_subset/subspace.hpp

```diff
@@ -50,15 +50,15 @@ class linear_subspace {
 template<typename FieldT>
 class affine_subspace : public linear_subspace<FieldT> {
 protected:
-    FieldT offset_;
+    FieldT shift_;
 
 public:
     affine_subspace() = default;
-    affine_subspace(const std::vector<FieldT> &basis, const FieldT &offset = FieldT(0));
-    affine_subspace(const linear_subspace<FieldT> &base_space, const FieldT &offset = FieldT(0));
-    affine_subspace(linear_subspace<FieldT> &&base_space, const FieldT &offset = FieldT(0));
+    affine_subspace(const std::vector<FieldT> &basis, const FieldT &shift = FieldT(0));
+    affine_subspace(const linear_subspace<FieldT> &base_space, const FieldT &shift = FieldT(0));
+    affine_subspace(linear_subspace<FieldT> &&base_space, const FieldT &shift = FieldT(0));
 
-    const FieldT& offset() const;
+    const FieldT& shift() const;
```

Comment:
Probably should drop the & here to unify with field_subset.hpp for now

Reply:
Yeah sounds good, will do.